sdp.go

```diff
@@ -55,6 +57,16 @@ func trackDetailsFromSDP(log logging.LeveledLogger, s *sdp.SessionDescription) m
 				}
 			}
 
+			// Handle a=msid for Unified plan
+			// TODO make "msid" a constant
+			if attr.Key == "msid" {
+				split := strings.Split(attr.Value, " ")
+				if len(split) == 2 {
+					trackLabel = split[0]
+					trackID = split[1]
+				}
+			}
+
 			if attr.Key == sdp.AttrKeySSRC {
```

Comment:
```go
switch attr.Key {
```
would be better.

Reply:
Thanks, I can modify it to use `switch()`. I'm just not sure if there are any guarantees that `a=msid` section will __always__ come before `a=ssrc`. If there are no guarantees, then we might have to keep a reference to last `trackDetails`, and update it when the `a=msid` section for the same track comes after `a=ssrc`.